content/articles/accessible-pdf-wheel.md

```diff
@@ -0,0 +1,229 @@
+the equivalent manner as an abled person.
+
+The content of an accessible PDF document needs to be in a _logical, or
+meaningful structure and sequential order_ regardless of its layout. This can
+be achieved using _PDF tags_. These tags provide assistive technologies, such
+as a screen reader, an organizational hierarchy of the document that can be
+read and tabbed through. These tags are similar to the semantic elements in
+HTML such as header, paragraph, image, and link.
+
+_The foundation of PDF accessibility is the tagging of its content._
+
+<div class="text-center text-black-50 my-5" aria-hidden="true"><i class="fas fa-lg fa-ellipsis-h"></i> <i class="fas fa-lg fa-file-pdf"></i> <i class="fas fa-lg fa-ellipsis-h"></i></div>
+
+This summer, the CTL launched its first massive open online course (MOOC)
+dedicated to the topic of inclusive teaching in higher education. The MOOC,
+titled [Inclusive Teaching: Supporting All Students in the College Classroom](https://www.edx.org/course/inclusive-teaching-supporting-all-students-in-the-college-classroom), is a self-paced course on edX, and is open
+to everyone. Accessibility was an extensive consideration for this MOOC, from
+its main video production to the development of its supplementary materials.
+
+## Challenge: Interactive Identity Wheel Activity
+
+One of the materials is the Identity Wheel Activity worksheet. The exercise
+encourages learners to identify and reflect on the different facets of their
+identities, and on how their identities are experienced differently in
+different contexts. The insight from this activity may help foster awareness on
+how these identities inform the way they understand the world, how they
+interact with it, and help them understand the idea of positionality.
+
+<p class="sr-only">The following image features a circle that is separated into
+12 sections. Each section is labeled: (starting at the top and moving clockwise
+around the circle) sexual identity; gender identity; race; ethnicity; religious
+affiliations; socioeconomic status; socioeconomic background; ability;
+educational background; national origin; first language; and family status. In
+the center of the circle, there are five numbered reflective prompts: (1)
+Identities you think about most often; (2) Identities you think about least
+often; (3) Your own identities you would like to know more about; (4)
+Identities that have the strongest effect on how you perceive yourself; (5)
+Identities that have the greatest effect on how others perceive you.</p>
+
+{{< figure
+    src="/img/assets/identity-wheel.png"
+    alt="The Identity Wheel Activity" >}}
+
+Each box on the wheel represents a different facet of identity. The learner
+will type in the textboxes the number that corresponds with each reflective
+prompts in the center of the circle as it relates to a particular identity
+facet. An interactive PDF seemed like a good fit for Identity Wheel worksheet
```

Comment:
Add `the`; `good fit for the Identity Wheel`

Reply:
Thanks